app/views/articles/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>
+  <strong>Title:</strong>
+  <%= @article.title %>
+</p>
+
+<p>
+  <strong>Text:</strong>
+  <%= @article.text %>
+</p>
+<h2>Article successfully created.</h2>
```

Comment:
Please use flash messages for this, not headings -- which go at the top of the file.  I recommend you use [.uk-alert](http://getuikit.com/docs/alert.html) for these